CONTRIBUTING.md

```diff
@@ -82,3 +81,29 @@ Before a pull request can be merged:
 * The feature branch must be rebased so it contains the latest changes from the target branch
 * The CI has to pass successfully
 * Every comment has to be addressed and resolved
+
+## Testing
+
+Tests are written using the *[Ginkgo](https://onsi.github.io/ginkgo/)* framework. Here are some general advices when writing tests:
+
+* Make sure using `Context` to specify what this test alludes to. Contexts should always start with _When_ (ie. "_When calling the function SomeFunction_")
```

Comment:
`s/Make sure using/Make sure to use/`

Reply:
thanks, @johnbieren. I thought  `using` and `to use` could be interchangeable. Anyway I went for the more wide change request from @davidmogar.

Reply:
Probably a good idea, he never likes what I type anyways ;)